src/__tests__/dataloader.test.js

```diff
@@ -889,36 +895,36 @@ describe('Accepts options', () => {
 
 describe('It allows custom schedulers', () => {
 
-  it('Supports manual dispatch', () => {
```

Comment:
does `batchScheduleFn` still work? it seems this was the only test that validated it. (the next test does not seem to use custom scheduling?)

Reply:
Yup, still works!

Reply:
I will add a test case